### ISSUE TITLE
[Docs] QStash docs still use v1 endpoint in CURL examples

### ISSUE THREAD

**Issue by miketromba** [upstash/issues#61]
See https://upstash.com/docs/qstash/features/deduplication#deduplication-id

![image](https://github.com/upstash/issues/assets/25141252/30fdb1f3-f646-4574-a844-0f242225278a)

**Comment by sancar** [comment]
Yes, this is intentional. We will change it to v2 when it is publicly announced. Thanks for the feedback.
